### PR TITLE
updated README to describe the repo

### PR DESCRIPTION
**Pull request by kristin-ng**
just updated my README to describe the repo! Very simple.

### CODE REVIEW

**Comment by kristin-ng** [comment]
i updated this with my personal portfolio done on the HTML/CSS/JS side!

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -1,28 +1,88 @@
...
+// // See the License for the specific language governing permissions and
+// // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+// /**
```
Comment (nitinms-sps-program) [inline portfolio/src/main/webapp/script.js:15]: Remove commented code.

**Review by nitinms-sps-program — APPROVED** [review]

**Comment by nitinms-sps-program** [comment]
Please send me a link to your deployed portfolio page.

**Review by nitinms-sps-program — APPROVED** [review]
Please merge changes into HEAD before making further changes. You have mixed up the servlet changes with last week's work. I am approving for now but please be careful in the future.

**Pull request merged**